.envbak

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
This file should not be committed, it's a redundant file (for backing up purpose) when we generate the .env file